Backpacks.cs

```diff
@@ -6586,25 +6588,24 @@ private void SetupContainer(ItemContainer container)
                 _backpackManager.RegisterContainer(container, this);
             }
 
-            private ItemContainer CreateContainerForPage(int page, int capacity)
+            private StorageContainer CreateContainerForPage(int page, int capacity)
             {
-                if ((object)_storageContainer == null || _storageContainer.IsDestroyed)
-                {
-                    _storageContainer = SpawnStorageContainer(0);
-                    if ((object)_storageContainer == null)
-                        return null;
-                }
+                var storageContainer = SpawnStorageContainer(capacity);
+                _backpackManager.RegisterContainer(storageContainer.inventory, this);
+                // if (page == 0)
+                // {
+                //     _storageContainer.inventory.capacity = capacity;
+                //     SetupContainer(_storageContainer.inventory);
+                //     return _storageContainer.inventory;
+                // }
+                
+                storageContainer.inventory.capacity = capacity;
+                SetupContainer(storageContainer.inventory);
```

Comment:
This line, and the `SetupContainer` method can be deleted, since that method has already been inlined in this method.